scripts/tests

```diff
@@ -239,6 +241,14 @@ _shared_options = [
         type=bool,
         help="Show verbose output or not",
     ),
+    click.option(
+        "-cp",
+        "--cross-prefix",
+        show_default=True,
+        default="aarch64-unknown-linux-gnu-",
+        nargs=1,
+        help="Cross prefix for compilation on non-aarch64 machines",
+    ),
```

Comment:
How do you handle that on MacOS you don't want to default to `aarch64-unknown-linux-gnu-`?

Reply:
Thanks for the catch!
I think it'd be better to set the options default to empty string for those who don't need this option, for example MacOS, to make it less confusing.